src/test/java/integration/ClientControllerIT.java

```diff
@@ -96,7 +99,7 @@ public void createClientAsIndividual() throws Exception {
         JSONObject inputJson = testHelper.getJsonObjectFromFile("json/client.json");
         inputJson.put("addressId", (int) newAddress.getId());
         inputJson.put("individual", true);
-
+        
```

Comment:
Plz, revert empty line without tab
Dima P. is always deleting it, so it's better if you don't change it